variable/test/operations_test.cpp

```diff
@@ -478,6 +478,19 @@ TEST(Variable, concatenate_unit_fail) {
   EXPECT_NO_THROW(concatenate(a, b, Dim::X));
 }
 
+TEST(Variable, concatenate_from_slices_with_broadcast) {
+  auto var =
+      makeVariable<double>(Dimensions{Dim::X, 4}, Values{0.0, 0.1, 0.2, 0.3});
```

Comment:
Suggest to add variances in the test, to ensure this also works (see comment above).